src/main/java/org/redcrosswarriors/emailservice/Request.java

```diff
@@ -0,0 +1,79 @@
+package org.redcrosswarriors.emailservice;
+
+/**
+ * this class holds data members for sending match notification emails
+ */
+public class Request {
+    private String name;
+    private String emailAddress;
+    private String phoneNumber;
+    private String message;
+
+
+    /**
+     * constructor - easiest way to build object
+     *
+     * @param name
+     * @param emailAddress
+     * @param phoneNumber
+     * @param message
+     */
+    public Request(String name, String emailAddress, String phoneNumber, String message) {
+        this.name = name;
+        this.emailAddress = emailAddress;
+        this.phoneNumber = phoneNumber;
+        this.message = message;
+    }
+
+    /**
+     * copy constructor
+     *
+     * @param _request
+     */
+    public Request(Request _request) {
```

Comment:
Please change _request to request. This is just so we follow Java's code style conformity. 
For future reference when programming in Java: 
All method names are in cammelCase. All class names are in PascalCase. 
All local variables and parameters should be in cammelCase.
constants should be in the form of: CONSTANT_NAME.